reference/index.yml

```diff
@@ -106,7 +106,7 @@ productDirectory:
     - title: Related technologies
       imageSrc: https://docs.microsoft.com/media/common/i_tools.svg
       links:
-        - url: https://docs.microsoft.com/powershell/wmf/
+        - url: https://docs.microsoft.com/powershell/scripting/wmf/
```

Comment:
fyi: link doesn't work, but is part of the change, correct?

Reply:
Fixed